website/docs/components/flyout/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,26 @@
+## Conformance
+
+**Note:** Putting this in for now, can adjust when the component is built.
```

Comment:
I would probably edit this note so it would make mores sense to any end user who may see the page prematurely. (Something along the lines of the API may shift as part of the Ember component development process?)

Reply:
To clarify here, the `hidden` property in frontmatter doesn't make the route accessible at all, not even from typing it in the browser manually. So I don't think there's a risk in anybody seeing it prematurely